src/components/loading/LoadingTableConnected.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { LoadingConnected } from './LoadingConnected';
+import { Loading } from './Loading';
+import * as _ from 'underscore';
+
+export class LoadingTableConnected extends LoadingConnected {
+  render() {
+    if (this.props.hide) {
+      return null;
+    }
+
+    return (
+      <table className='mod-collapsible-rows'>
+        {_.range(this.props.numberOfRows).map((num: number) => {
+          return (
+            <tbody key={`loading-row${num}`} className='loading-row'>
+              <tr>
+                <td colSpan={this.props.columnsPerRow}>
+                  <Loading />
```

Comment:
Shouldn't it be one loading per table instead of one loading per row?

Reply:
as said earlier, this will be removed, it's ugly anyways :D